src/chocolatey.tests.integration/scenarios/PackScenarios.cs

```diff
@@ -115,6 +115,54 @@ public void sources_should_be_set_to_specified_output_directory()
             }
         }
 
+        [Concern(typeof(ChocolateyPackCommand))]
+        public class when_packing_with_properties : ScenariosBase
+        {
+            private readonly string package_path = Path.Combine(Scenario.get_top_level(), "test-package.0.1.0.nupkg");
+
+            public override void Context()
+            {
+                base.Context();
+
+                Scenario.reset(Configuration);
+
+                Configuration.PackCommand.Properties.Add("version", "0.1.0");
```

Comment:
Strange that you added version - it is already a first class citizen - should `--version 1.2.3` override `version=1.2.3` if both are specified?